skbio/stats/distance/_permdisp.py

```diff
@@ -140,7 +140,7 @@ def permdisp(distance_matrix, grouping, column=None, test='median',
     sample size                      6
     number of groups                 2
     test statistic             1.03296
-    p-value                       0.35
+    p-value                       ...
```

Comment:
@ElDeveloper, this should be deterministic, right?

Reply:
Indeed, although I believe different versions of scipy/numpy have yielded a few different values that are all within a few decimal places and wouldn't affect the significance of the results. I believe this might boil down to implementation discrepancies between versions.

Any suggestions on how to handle this? I would prefer to mask that value and just refer to it as significant/not significant but not sure if that's a good idea.

Reply:
I don't think I have seen other scikit-packages embed random numbers within their docs (see the scipy page on [norm](https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.norm.html) for instance). If there is a function that relies on random numbers, histograms or asymptotic stats are used. It very well could be that different machines have different random number generators.  I'd suggest one of the 2 options

1. Use more permutations - even if the rng is different, it should converge to the right answer with enough permutations.
2. Ignore this doctest and push the test for pvalues into the unittests.  This would be my suggestion, since functions relying on rngs will lead to inherently unstable tests - best to sanity check the asymptotic behavior in a unittest.